rise/static/main.js

```diff
@@ -420,24 +435,14 @@ define([
 
               Reveal.initialize();
 
-              var options = {
-                // All this config option load correctly just because of require-induced delay,
-                // it would be better to catch them from the config.get promise.
-                controls: config.get_sync('controls'),
-                progress: config.get_sync('progress'),
-                history: config.get_sync('history'),
+              // all these settings are passed along to reveal as-is
+              // xxx it might be just better to copy the whole complete_config instead
+              // of selecting some names, which would allow users to transparently use
+              // all reveal's features
```

Comment:
I agree but that is some work to be done later... not in this PR.

Reply:
totally agree, it's just a note to not forget to consider that option at some later point

Reply:
Yep.